Rakefile

```diff
@@ -49,17 +49,11 @@ namespace :etld_data do
     sh 'git', 'commit',
       ETLD_DATA_FILE,
       ETLD_DATA_RB,
-      '-m', 'Update the eTLD database to %s.' % DomainName::ETLD_DATA_DATE
+      '-m', 'Update the eTLD database to %s.' % DomainName.etld_data_date
   end
 end
 
-file ETLD_DATA_RB => [
-  ETLD_DATA_FILE,
-  ETLD_DATA_RB + '.erb',
-  'tool/gen_etld_data.rb'
-] do
-  ruby 'tool/gen_etld_data.rb'
-end
+ruby 'tool/gen_etld_data.rb'
```

Comment:
Yeah, I guess I did not understand completely the previous implementation. Changing that now to revert that.